ios/FluentUI/List/ListItem.swift

```diff
@@ -47,42 +47,60 @@ public struct ListItem<LeadingContent: View,
         tokenSet.update(fluentTheme)
 
         @ViewBuilder
-        var labelStack: some View {
-            let titleView = Text(title)
-                                .foregroundColor(Color(uiColor: tokenSet[.titleColor].uiColor))
-                                .font(Font(tokenSet[.titleFont].uiFont))
-                                .frame(minHeight: ListItemTokenSet.titleHeight)
-                                .lineLimit(titleLineLimit)
-                                .truncationMode(titleTruncationMode)
-                                .accessibilityIdentifier(AccessibilityIdentifiers.title)
+        var titleView: some View {
+            Text(title)
+                .foregroundColor(Color(uiColor: tokenSet[.titleColor].uiColor))
+                .font(Font(tokenSet[.titleFont].uiFont))
+                .frame(minHeight: ListItemTokenSet.titleHeight)
+                .lineLimit(titleLineLimit)
+                .truncationMode(titleTruncationMode)
+                .accessibilityIdentifier(AccessibilityIdentifiers.title)
+        }
+
+        @ViewBuilder
+        var subtitleView: some View {
+            let subtitleView = Text(subtitle)
+                .foregroundColor(Color(uiColor: tokenSet[.subtitleColor].uiColor))
+                .lineLimit(subtitleLineLimit)
+                .truncationMode(subtitleTruncationMode)
+                .accessibilityIdentifier(AccessibilityIdentifiers.subtitle)
 
             switch layoutType {
-            case .oneLine:
-                titleView
-            case .twoLines, .threeLines:
-                let subtitleView = Text(subtitle)
-                    .foregroundColor(Color(uiColor: tokenSet[.subtitleColor].uiColor))
-                    .lineLimit(subtitleLineLimit)
-                    .truncationMode(subtitleTruncationMode)
-                    .accessibilityIdentifier(AccessibilityIdentifiers.subtitle)
-                VStack(alignment: .leading, spacing: ListItemTokenSet.labelVerticalSpacing) {
+            case .oneLine, .twoLines:
+                subtitleView
```

Comment:
this technically never gets used for oneLine. is there a way to assert or return an empty view